### PR TITLE
Basic Tutorial Example - Django bims 525

### PR DESCRIPTION
**Pull request by Christiaanvdm**
This is the version 1 tutorial - Serves as an example for FBIS to draw up a more complete tutorial for.

This should not close the ticket.

django-bims-525v1
 -PS: Sorry for the delay in uploading the GIF, my upload was running for very long earlier and didn’t want to finish.

### CODE REVIEW

**Comment by dimasciput** [comment]
Can you show screenshots?

File: bims/templates/main_base.html [hunk 1]
```diff
@@ -39,10 +42,12 @@
 {% block body_content %} {% endblock %}
 
 <!-- Bootstrap core JavaScript -->
+<!-- IntroJS -->
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/intro.js/2.9.3/intro.min.js"></script>
+    <script type="text/javascript" src="../static/js/views/introjs-tutorial.js"></script>
```
Comment (dimasciput) [inline bims/templates/main_base.html]: use `{% static ... to load the static file

File: bims/static/js/views/introjs-tutorial.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                element: "#download-control",
+                intro: "We are not stingy with our data either. DOWNLOAD...",
+                position: 'right'
+              },
+                 {
+                element: "#print-control",
+                intro: "...or PRINT, exactly what you need, quickly.",
+                position: 'right'
+              },
+              // {
```
Comment (dimasciput) [inline bims/static/js/views/introjs-tutorial.js]: If we don't need it, just remove it

File: bims/templates/map_page/map-templates.html [hunk 3]
```diff
@@ -28,7 +39,8 @@
         </div>
         <div class="right-panel-header">
             <div class="pull-right">
-                <img class="close-icon close-panel header-panel-button" src="/static/img/close_icon.svg">
+                <img class="close-icon close-panel header-panel-button"
```
Comment (dimasciput) [inline bims/templates/map_page/map-templates.html]: Next time please don't reformat the code, it's hard to check which changes.

Comment (Christiaanvdm) [reply bims/templates/map_page/map-templates.html]: Roger

File: bims/static/js/views/introjs-tutorial.js (hunk 2 above)
Comment (Christiaanvdm) [reply bims/static/js/views/introjs-tutorial.js]: Fixed.

File: bims/templates/main_base.html (hunk 1 above)
Comment (Christiaanvdm) [reply bims/templates/main_base.html]: Fixed.

**Pull request merged**